products/sle/main.pm

```diff
@@ -502,7 +507,7 @@ sub load_inst_tests() {
     }
     loadtest "installation/addon_products_sle.pm";
     if (noupdatestep_is_applicable) {
-        if (check_var('ARCH', 'x86_64') && sle_version_at_least('12-SP2') && is_server && (!is_sles4sap || is_sles4sap_standard)) {
+        if (check_var('ARCH', 'x86_64') && sle_version_at_least('12-SP2') && is_server && (!is_sles4sap || is_sles4sap_standard) && !check_var('NOT_INSTALL_TO', 1)) {
```

Comment:
then please move this into this check into a more speaking function.

E.g. 
```
sub not_installing_this_version {
  !check_var('NOT_INSTALL_TO', 1)
}
```
And use this function here instead. Beside that I consider a NOT variable defaulting to 1 rather confusing.